src/content/get-started/advanced-commands-and-concepts.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Advanced Commands & Concepts for Developers
+description: Introduction to development concepts within Okteto
+sidebar_label: Advanced Commands and Concepts
+id: advanced-commands-and-concepts
+---
+
+import Image from '@theme/Image';
+
+## Advanced CLI Commands
+
+| Command          | Description                                                                                                                 |
+| ---------------- | --------------------------------------------------------------------------------------------------------------------------- |
+| okteto namespace | Allows you to deal with namespaces (create, delete, use).                                                                   |
```

Comment:
Should we add links to the full docs of each command?

Reply:
We can also do something like at the end of this section:

"If you want to learn more about any of these commands in details, checkout the complete [Okteto CLI reference](link)"